src/commcare_cloud/fab/fabfile.py

```diff
@@ -386,11 +386,11 @@ def prepare_offline_deploy():
 
 @task
 def setup_limited_release(keep_days=1):
-    setup_release(keep_days, False)
+    setup_release(keep_days, 'False')
```

Comment:
nit: I'd name this parameter so you can read this line and know what `'False'` means - in general I'd almost always prefer to name boolean arguments, unless it's clear from the function name what it's for.

Reply:
Seems reasonable. My standard also generally includes if the function definition is right next to it, but it's probably still better to name it anyway